docs/DirectManipulation.md

```diff
@@ -83,11 +83,11 @@ performing continuous animations and responding to gestures, judiciously
 optimizing your components can improve your animations' fidelity.
 
 If you look at the implementation of `setNativeProps` in
-[NativeMethodsMixin.js](https://github.com/facebook/react-native/blob/master/Libraries/ReactIOS/NativeMethodsMixin.js)
+[NativeMethodsMixin.js](https://github.com/facebook/react/blob/master/src/renderers/native/NativeMethodsMixin.js)
```

Comment:
Yeah, it's ok to leave it more generic (without a line number), I think. We do provide the method name `setNativeProps` above, so that is good.